docs/docs/routing.md

```diff
@@ -0,0 +1,238 @@
+---
+id: routing
+title: Routing
+permalink: docs/routing.html
+---
+
+Routing is the process of choosing what to render based on the browser's current URL.
+
```

Comment:
Let's make another intro paragraph to explain that React itself doesn't provide any built-in routing solutions, but you can use component lifecycle and state (link it to the doc) to hook it up to the browser APIs.